### PR TITLE
[BFD-821] moves some items to vaults

### PR DESCRIPTION
**Pull request by keithdadkins**
### Change Details
Moves items with potentially sensitive data into vault encrypted files.

### Acceptance Validation
- [x] branch deployed to test environment

### External References

- [BFD-xxx](https://jira.cms.gov/browse/BFD-xxx)

### Security Implications

- [ ] new software dependencies

- [ ] altered security controls

- [ ] new data stored or transmitted

- [ ] security checklist is completed for this change

- [ ] requires more information or team discussion to evaluate security implications

### CODE REVIEW

File: ops/ansible/playbooks-ccs/vars/test/group_vars/all/env_specific.yml [hunk 1]
```diff
@@ -3,12 +3,6 @@
 # Version of the ETL Jar to deploy 
 data_pipeline_version: '0.1.0-SNAPSHOT'
 
-# Bucket where deployment artifacts are located. 
-s3_bucket_deployment: 'bfd-mgmt-artifacts-577373831711'
```
Comment (rtravitz) [inline ops/ansible/playbooks-ccs/vars/test/group_vars/all/env_specific.yml:7]: The test vault also has this value listed as `vault_s3_bucket_deployment`, but none of the other envs have it. Any reason for that?

**Review by rtravitz — APPROVED** [review]
I see their inclusion in all three vaults.

File: ops/ansible/playbooks-ccs/vars/test/group_vars/all/env_specific.yml (hunk 1 above)
Comment (njdister) [reply ops/ansible/playbooks-ccs/vars/test/group_vars/all/env_specific.yml:7]: It looks to have existed there before this PR, per the master branch.  Lack of diff in vault is rough  :/

**Review by njdister — APPROVED** [review]
I believe these 2 values s3_bucket_deployment and s3_bucket_etl_staging are unused but it doesn't hurt to at least move them out of sight to remediate issues.

**Pull request merged**